<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationViewer</class>
 <widget class="QWidget" name="AnimationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animation Viewer</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="delayLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>13</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames / second :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="delayComboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as AVI</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
